internal/common/rules/ExampleMatrixRegistration.go

```diff
@@ -0,0 +1,30 @@
+package rules
+
+import (
+	"gonum.org/v1/gonum/mat"
+)
+
+func RegisterCoolRool() {
+
+	//A very contrived rule//
+
+	registerNewVariable("number_of_islands_contributing_to_common_pool", 5)
+	registerNewVariable("number_of_failed_forages", 0.5)
+	registerNewVariable("number_of_broken_agreements", 1)
+	registerNewVariable("max_severity_of_sanctions", 2)
+
+	name := string("Kinda Complicated Rule")
```

Comment:
i think you can skip the `string(...)`

Reply:
Lol GoLand warned me and I did not listen, removing that now